### ISSUE TITLE
hyper threading

### ISSUE THREAD

**Issue by cheers00** [TechEmpower/FrameworkBenchmarks#1768]
Are you running the techempower benchmarks with Hyper-Threading enabled?

**Comment by zloster** [comment]
Better late than never:

- For the physical hardware you can check the `dstat` command outputs for the given benchmark. For example `dropwizard` round 11 test run data is [here](https://github.com/TechEmpower/TFB-Round-11/blob/master/peak/linux/2015-09-11-final/20150911140159/json/dropwizard/stats#L6). Note the columns labeled `cpu0 usage`, `cpu1 usage`...`cpu39 usage`. The number of reported cores is the same as stated in the [environment details](https://www.techempower.com/benchmarks/#section=environment) 2 x [Xeon E5-2660 v2](http://ark.intel.com/products/75272) with 20 hyper-threading cores each.
- For Amazon EC2 you don't have choice: the following information is just after the [Instance Types Matrix table](http://aws.amazon.com/ec2/instance-types/). 

> Each vCPU is a hyperthread of an Intel Xeon core for M4, M3, C4, C3, R3, HS1, G2, I2, and D2.

So basically every `vCPU` you can rent on Amazon from their current generation is a hyper-threading core (excluding the `t2` instances which are not specified).

**Comment by msmith-techempower** [comment]
tl;dr - yes
